test/utils/test_assortativity.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+import torch
+
+from torch_geometric.utils import assortativity, coalesce
+
+
+def test_assortativity():
+    torch.manual_seed(12345)
+
+    num_nodes = 100
+    num_edges = 1000
+    edge_index = torch.randint(0, num_nodes, size=(2, num_edges))
+    edge_index = coalesce(edge_index)
+    out = assortativity(edge_index)
+    assert pytest.approx(out, abs=1e-5) == -0.05669079348444939
```

Comment:
Its not totally clear what this is testing to me? Is it possible for us to have a fixed example where it is clear what that `assortativity` value should be?

Reply:
Got it. Let me fix it later.